src/main.css

```
@@ -286,13 +286,16 @@ body {
 
 .dynmap .leaflet-control-container .leaflet-minimap-container[class*="the_end"] .world .item .maptype,
 .dynmap .leaflet-control-container .leaflet-minimap-container[class*="DIM1"] .world .item .maptype,
+.dynmap .leaflet-control-container .leaflet-minimap-container[class*="nullscape_end_island"] .world .item .maptype,
```

Great! I'll note Nullscape support in the changelog, but a more generic solution would be good for the MC community here:
```suggestion
.dynmap .leaflet-control-container .leaflet-minimap-container:is([class^="end_"], [class*="_end_"]) .world .item .maptype,
```